classes/lrucache.js

```diff
@@ -0,0 +1,71 @@
+class LRUCache {
+  constructor (max = 0) {
```

Comment:
If we are not exporting this, and we are not exposing a way for end users to change this value, there's no need for `max` to be anything other than a fixed value.

Reply:
We also wouldn't need a type check that is only relevant in the unit test, and not in any real world use case.

Reply:
done